lib/screens/event/card.dart

```diff
@@ -0,0 +1,93 @@
+import 'package:Solon/util/user_util.dart';
+import 'package:flutter/material.dart';
+
+import 'package:Solon/screens/event/page.dart';
+import 'package:Solon/util/app_localizations.dart';
+import 'package:Solon/widgets/screen_card.dart';
+import 'package:Solon/models/event.dart';
+import 'package:Solon/services/event_connect.dart';
+import 'package:Solon/widgets/text_layout.dart';
+
+class EventCard extends StatefulWidget {
+  final Event event;
+
+  EventCard({Key key, this.event}) : super(key: key);
+
+  @override
+  _EventCardState createState() => _EventCardState();
+}
+
+class _EventCardState extends State<EventCard> {
+  Future<bool> getAttendanceVal() async {
+    final sharedPrefs = await UserUtil.connectSharedPreferences(key: 'userData');
+    final userUid = sharedPrefs['uid'];
+    final responseMessage = await EventConnect.getAttendance(
+      eid: widget.event.eid,
+      uid: userUid,
+    );
+    return responseMessage;
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    Function function = () {
+      Navigator.push(
+        context,
+        MaterialPageRoute(
+          builder: (context) => EventPage(
+            event: widget.event,
+          ),
+        ),
+      );
+    };
+    ListTile tile = ListTile(
+      contentPadding: EdgeInsets.only(
+        top: 10,
+        bottom: 10,
+        right: 15,
+        left: 15,
+      ),
+      title: Padding(
+        padding: const EdgeInsets.only(bottom: 8),
+        child: TextLayout.fillLinesWithTextAndAppendTrail(
+          rawText: widget.event.title,
+          trail: '...',
+          textStyle: TextStyle(
+            fontFamily: 'Raleway',
+            fontWeight: FontWeight.bold,
+            fontSize: 22,
```

Comment:
abstract styles to another file

Reply:
where do you suggest we put a potential styles file

Reply:
a styles folder would be better since there are other styling widgets other than textstyle; since styles are part of our app code, it should be in lib/